core/src/main/java/org/openconnectors/connect/Connector.java

```diff
@@ -54,21 +45,9 @@ public void initialize(final ConnectorContext ctx){
     public abstract void open(final Config config) throws Exception;
 
     /**
-     * Reset the connector with a new configuration
+     * Close the connector
      *
-     * @param config
      * @throws Exception
      */
-    public void reset(final Config config) throws Exception{
-        close();
-        open(config);
-    }
-
-    public ConnectorContext getContext() {
-        return context;
-    }
-    public AtomicBoolean isOpen() {
-        return isOpen;
-    }
-
+    public void close() throws Exception;
```

Comment:
AutoCloseable provides this interface

Reply:
I know. I personally think having it explicit is better. But could go either way

Reply:
I prefer the AutoCloseable, it allows us to use the modern try get

Reply:
will change

Reply:
changed